### PR TITLE
Required changes for CASP VMX images

### PR DESCRIPTION
**Pull request by kravciak**
Enable second HDD device in sshVirsh.
Use SERIALDEV variable if set

### CODE REVIEW

File: testapi.pm [hunk 1]
```diff
@@ -99,11 +99,11 @@ sub init {
...
     }
     elsif (check_var('VIRSH_VMM_FAMILY', 'xen') && check_var('VIRSH_VMM_TYPE', 'linux')) {
-        if (check_var('VERSION', '12-SP2')) {
-            $serialdev = "hvc0";
+        if (check_var('VERSION', '12-SP1')) {
+            $serialdev = "xvc0";
         }
         else {
-            $serialdev = "xvc0";
+            $serialdev = "hvc0";
```
Comment (okurz) [inline testapi.pm]: why is it necessary to turn this around now? Was it never tested in before?

Comment (kravciak) [inline testapi.pm]: I can use "check_var('VERSION', '12-SP2') or is_casp" or turn it around - this way it will work also for SP3 + ...seems cleaner

Comment (okurz) [inline testapi.pm]: Actually I don't quite get why we need to distinguish between SLE versions in the testapi. That is pretty wrong. Maybe you want to consult with @mnowaksuse and come up with an approach where the backend does not need to know anything about SLE. Maybe some `$distri` method can help here, e.g. as is done for `script_run` and such.

Comment (coolo) [inline testapi.pm]: this code really needs to move to main.pm and set SERIALDEV. If that doesn't work, we need to make it work.

**Pull request merged**